Formula/sphinx.rb

```diff
@@ -19,6 +19,12 @@ class Sphinx < Formula
     sha256 x86_64_linux:   "3736780f2809b0449953cfb4ed85cc7d65b86e6c5c55aa98186062f8aff81d99"
   end
 
+  # Some CVEs and corresponding Debian patches:
+  # - CVE-2019-14511: https://sources.debian.org/patches/sphinxsearch/2.2.11-8/config-default-to-localhost.patch/
+  # - CVE-2020-29050: https://sources.debian.org/patches/sphinxsearch/2.2.11-8/06-CVE-2020-29050.patch/
+  # Ref: https://github.com/sphinxsearch/sphinx#sphinx
+  deprecate! date: "2022-08-15", because: "is using unsupported v2 and source for v3 is not publicly available"
```

Comment:
Checking for feedback as I wasn't sure if this is best approach given that upstream is supposedly going to open-source v3. Though, not sure if there is any timeframe as v2 was claimed unsupported in 2018.

We could patch CVEs like Debian/Fedora and try to keep v2 until v3 source is available.

Reply:
Let's deprecate it, but apply the patches for users who currently have it installed or will install it before this gets disabled.